bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -322,8 +322,25 @@ const isMeetingLocked = (id) => {
 };
 
 const setEmojiStatus = (s) => {
-  const status = s === Auth.userID ? 'none' : s;
-  makeCall('setEmojiStatus', Auth.userID, status);
+  switch (s) {
```

Comment:
import `EMOJI_STATUSES` and ` Object.keys(EMOJI_STATUSES).includes(s) ? s : 'none'`